src/components/Markdown/Markdown.tsx

```diff
@@ -29,7 +29,7 @@ interface MarkdownProps {
 
 interface MarkdownState {}
 
-const md = markdown_it({
+const md = markdownit({ // cspell:disable-line
```

Comment:
Prettier won't allow this, it needs to be `// cspell: disable-next-line` on the previous line .